aws_lambda_powertools/utilities/data_classes/common.py

```diff
@@ -10,6 +10,12 @@ def __init__(self, data: Dict[str, Any]):
     def __getitem__(self, key: str) -> Any:
         return self._data[key]
 
+    def __eq__(self, other: Any) -> bool:
+        if not isinstance(other, DictWrapper):
+            return NotImplemented
```

Comment:
```suggestion
            return False
```

Shouldn't the `__eq__` method just return `False` instead of a `NotImplemented`? If the Type is not a `DictWrapper` it's never equal hence it should return False?

Reply:
It should indeed return a bool not an Exception.